src/components/HomePage/SearchArea/useUrlSearchParams.js

```diff
@@ -128,6 +128,39 @@ export default ({
 
     }, [clearURLFilters, location, resetAdvancedSearchFields]);
 
+    const setUrlFilters = useCallback((
```

Comment:
Why dos you create e new separarem function for this? Couldn't you have used `changeUrlFilters` ?

Reply:
I think I used it to reset, but I honestly can't remember now. I will test using `changeUrlFilters`